addons/account/models/account_move.py

```diff
@@ -951,11 +952,9 @@ def _compute_secured(self):
             move.secured = bool(move.inalterable_hash)
 
     def _search_secured(self, operator, value):
-        if operator not in ['=', '!='] or value not in [True, False]:
-            raise UserError(_('Operation not supported'))
-
-        want_secured = (operator == '=') == value
-        return [('inalterable_hash', '!=' if want_secured else '=', False)]
+        if operator != 'in':
+            return NotImplemented
+        return [('inalterable_hash', '!=', False)]
```

Comment:
`('secured', '=', True)` and `('secured', '=', False)` should not be the same

Reply:
For boolean fields, there is an optimization that ensures that value is `[True]` which eases the implementation of search methods of boolean fields. I will add an assert for documentation purposes.